draft-ietf-quic-transport.md

```diff
@@ -703,6 +703,22 @@ use it for all subsequent Handshake ({{packet-handshake}}) and 1-RTT
 Server's Version Negotiation ({{packet-version}}) and Retry ({{packet-retry}})
 packets MUST use connection ID selected by the client.
 
+A connection ID is truncated when included in short headers.  Each endpoint
+expresses how many octets of the connection ID they require in packets that are
+sent to that endpoint.  The truncate_connection_id transport parameter
+({{transport-parameter-definitions}}) determines the number of octets that are
+included.  An endpoint can request that the connection ID to be omitted entirely
+by setting truncate_connection_id to 0, or include any number of octets up to
+the full connection ID length of 17 octets.  Connection IDs are not truncated
+for any other use.
+
+Note:
+
+: The odd length of the connection ID is chosen to enable efficient construction
+  of the value using 128-bit ciphers with additional space for overhead.
+  Endpoints might negotiate the use of a truncated connection ID for use with
+  the short header.
```

Comment:
The second sentence of the note seems pointless, given that you've just talked about this and it's unrelated to the first sentence in the note.